_data/simple-icons.json

```
@@ -6133,6 +6133,12 @@
             "hex": "F2CC38",
             "source": "https://about.glovoapp.com/press"
         },
+        {
+            "title": "glTF",
+            "hex": "87c540",
```

```suggestion
            "hex": "86C540",
```
This is what I'm seeing from their logo assets zip.

---

https://www.khronos.org/files/legal/Khronos-Logo-Family-Color-Guide.pdf

---

Ah looks like the logo usage PDF uses `#87C540?`

---

We'll just stick with the logo usage guide / what you got already.